tests/e2e/specs/out-of-game/localization.spec.js

```diff
@@ -0,0 +1,37 @@
+import { myUser } from '../../fixtures/userFixtures';
+import es from '../../../../src/translations/es.json';
+import fr from '../../../../src/translations/fr.json';
+
+function setup() {
+  cy.viewport(1920, 1080);
+  cy.wipeDatabase();
+  cy.visit('/');
+  cy.signupPlayer(myUser);
+  cy.vueRoute('/');
+}
+
+describe('Localization ', () => {
+  beforeEach(setup);
+
+  it('Checking Translation For Spanish', () => {
+    langaugeMenu('es', es);
+  });
+
+  it('Checking Translation For French', () => {
+    langaugeMenu('fr', fr);
+  });
+});
+
+function langaugeMenu(name, data) {
+  // Open the menu
+  cy.get('[data-cy="user-menu"]').click();
+  cy.get('[data-cy="language-menu"]').click();
+
+  //Selects the language 
+  cy.get('[data-lang="' + name + '"]').click();
+
+  // checks elements in navbar has changed language
+  cy.get('[data-cy="' + data.global.play + '"]').should('contain', data.global.play);
+  cy.get('[data-cy="user-menu"]').click();
+  cy.get('[data-nav="' + data.global.logout + '"]').should('contain', data.global.logout);
+}
```

Comment:
Yes because in this 
```
const menuItems = computed(() => {
  return [{ text: t('global.logout'), icon: 'logout', page: { name: ROUTE_NAME_LOGOUT } }];
});
```
this text is passing in data-nav so it is updating when language is changed.

Reply:
Honestly I’d leave the data selectors using the locale abbreviations so it’s not even something we need to translate at all and more consistent for testing.

Reply:
yes i did that in recent commit can you verify once.